src/App.jsx

```diff
@@ -8,7 +8,7 @@ export const App = () => {
   return (
     <Router>
       <Switch>
-        <Route path="/auth">
+        <Route path="/:access_token(access_token=.*)">
```

Comment:
Here I use some kind of regular expression, as I was suggested at stackowerflow, but I do not understand it.

Reply:
It looks like unnecessary complication. Your app does not really need that route (as it's just for callback - not for the user). 
I would suggest to remove that route, make auth callback to the root (/) and parse querystring as an effect